packages/victory-native/src/components/victory-container.js

```diff
@@ -0,0 +1,156 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Svg, { Rect } from "react-native-svg";
+import { assign, get } from "lodash";
+import { View, PanResponder } from "react-native";
+import { VictoryContainer } from "victory-core/es";
+import NativeHelpers from "../helpers/native-helpers";
+import Portal from "./victory-portal/portal";
+
+const yes = () => true;
+const no = () => false;
+
+export default class extends VictoryContainer {
```

Comment:
TODO: we should investigate getting away from class extension here as well.

Reply:
Agreed! There were a few places where there's class extension, but in a non-trivial way, and tackling those will require a little more attention. I figured those should probably happen in their own unit of work to ensure the conversion/refactoring doesn't come with any regressions.